src/sentry/models/integration.py

```diff
@@ -171,7 +171,4 @@ def reauthorize(self, data):
         Each provider may have their own way of reauthorizing the
         integration.
         """
-        if self.provider == "slack":
```

Comment:
@MeredithAnya i think we can just remove this method and where it's used, no?

Reply:
yeah i wasn't sure if we should keep it if we needed to reauth in the future but like im removing all the frontend code and can always re-implement so yeah ill take that out